### ISSUE TITLE
Posts made with Netlify not showing up

### ISSUE THREAD

**Issue by sfkpmr** [thiagorossener/jekflix-template#72]
Trying out Netlify and have made some test posts with their CMS, but they don't show up most of the time.

I've tried entering data in all data fields, subtitle, description, etc.

Just now I did a test when only two of four posts were being displayed, and I removed the subtitle for one of the posts that was being displayed, and suddenly the two remaining ones showed up too.

All posts are in the _/posts folder in my repo.

The posts are published, and the latest update is published in Netlify.

I'm not seeing any obvious differences between what's in my repo now, and what I've seen when running Jekyll on my own server.

**Comment by thiagorossener** [comment]
Hey @sfkpmr, first of all, sorry for the delay.

It seems to be an issue with Netlify. Do you have a link for your website and your repo?

Bear in mind that even if you're changing your posts locally, what Netlify shows is always what is in the repo and in the branch set up in `admin/_config.yml` (which is usually `master`).

Another thing to pay attention is that Netlify takes a while to build the site, so it might have been queued and when you updated again, it finished building the previous changes (just a guess).

**Comment by sfkpmr** [comment]
Thanks for getting back to me! :)

Yes, an issue with Netlify would probably make most sense considering it worked great locally.

The site is on www.wallen.co.

I haven't looked at this since the 10th of March, and when I went to the site now it showed two posts. I added an "April test" post, but instead of showing that it pushed a "Hello world" post. Total of 3 posts showing now. Then I made another "Second April test" post, and nothing changed.

**Comment by thiagorossener** [comment]
Yeah, I've just looked into your repo, the posts look good. You probably need to check those Netlify's logs.

Do all of them show up when running `gulp` locally?

**Comment by sfkpmr** [comment]
I don't have it running locally at the moment, but as it worked fine before
I'm sure it would again. Wasn't using Netlify then of course.

**Comment by chaitanyasaxena066** [comment]
@sfkpmr Hey Is your issue is solve or still it is not deploying as same issue is also occurring with me I also can't able to see my updated posts on my website

**Comment by sfkpmr** [comment]
Still not showing up.

**Comment by sfkpmr** [comment]
Decided to delete the repo and fork it again. The sample posts show up, but like before, I can't make new posts.

**Comment by thiagorossener** [comment]
Hmm I really have no idea what could possibly be going on. Can you post the Netlify log? If you were able to grab a link for the Netlify log would be great.

I'm not sure if it's possible, but for example, clicking on the Netlify status button in the README file, it takes me to the demo blog log:
https://app.netlify.com/sites/sleepy-bhabha-00eedf/deploys

**Comment by sfkpmr** [comment]
I asked for help on the Netlify board too, so I'm gonna try their troubleshooting steps when I get a chance!

**Comment by Party4Bread** [comment]
I have same problem too.
Seems like gulp can't detect new .md file

**Comment by troke12** [comment]
@thiagorossener i have same problem with this too, when you are making a new post in current date, they are not really showing up, the better solutions that i found it's make that `date:` in posts are not in current time, it should be show up

**Comment by tuminzee** [comment]
> Trying out Netlify and have made some test posts with their CMS, but they don't show up most of the time.
> 
> I've tried entering data in all data fields, subtitle, description, etc.
> 
> Just now I did a test when only two of four posts were being displayed, and I removed the subtitle for one of the posts that was being displayed, and suddenly the two remaining ones showed up too.
> 
> All posts are in the _/posts folder in my repo.
> 
> The posts are published, and the latest update is published in Netlify.
> 
> I'm not seeing any obvious differences between what's in my repo now, and what I've seen when running Jekyll on my own server.

I have the same issue
